How-GitHub-Issue-Moderators-AKA-Issue-Mods-Work.md

```diff
@@ -73,4 +73,4 @@ The number of issue moderators will always remain small due to the nature of Git
 * Non-Default Profile Image set on GitHub
 
 #### Activity Requirement
-Please note that we will frequently remove issue mods whom we think are inactive. If you are removed, please do not take this personally - we can add you back to the team, just message us and let us know you're still active.
+Please note that we will frequently remove issue mods who we think are inactive. If you are removed, please do not take this personally - we can add you back to the team, just message us and let us know you're still active.

```

Comment:
It'd instead be "whom we think to be inactive", but I'd advice against using this, for it's a disused form from the Anglicisation of the Latin accusative-and-infinitive construction.